desktop/hardened-desktop/admin-controls/configure-ac.md

```diff
@@ -0,0 +1,129 @@
+
+The following `admin-settings.json` code and table provides the required syntax and descriptions for parameters and values:
+
+```json
+{
+  "configurationFileVersion": 2,
+  "exposeDockerAPIOnTCP2375": {
+    "locked": true,
+    "value": false
+  },
+  "proxy": {
+    "locked": false,
+    "mode": "system",
+    "server": "myproxy.com",
+    "port":3129,
+    "exclude": ["foo.com", "bar.com"]
+  },
+  "enhancedContainerIsolation": {
+    "locked": false,
+    "value": false
+  },
+  "useWindowsContainers": {
+      "locked": false,
+      "value": false
+  },
+  "linuxVM": {
+    "wslEngineEnabled": {
+      "locked": false,
+      "value": false
+    },
+    "dockerDaemonOptions": {
+      "locked": false,
+      "value":"<json string>"
+    },
+    "vpnkitCIDR": {
+      "locked": false,
+      "value":"192.168.65.0/24"
+    }
+  },
+  "windowsContainers": {
+    "dockerDaemonOptions": {
+      "locked": false,
+      "value":"<json string>"
+    }
+  },
+  "disableUpdate": {
+    "locked": false,
+    "value": false
+  },
+  "analyticsEnabled": {
+    "locked": false,
+    "value": true
+  },
+}
+```
+
+| Parameter                        | Description                      |
+| :------------------------------- | :------------------------------- |
+| `configurationFileVersion`        | Specifies the version of the configuration file format.    |
+| `exposeDockerAPIOnTCP2375` |<span class="badge badge-info">Windows only</span> Exposes the Docker API on a specified port. If `value` is set to true, the Docker API is exposed on port 2375. Note: This is unauthenticated and should only be enabled if protected by suitable firewall rules.|
+| `proxy` | It is used for `http` and `https`.  If the port is custom, specify it in the property. |
+| `enhancedContainerIsolation`  | If `value` is set to true, Docker Desktop prevents user containers from running as root and from being able to mount sensitive Docker Desktop configuration directories from the Docker Desktop VM. |
+|`useWindowsContainers` | <span class="badge badge-info">Windows only</span> If `value` is set to true, it switches Docker Desktop to toggle the Docker CLI to talk to the Windows daemon, enabling Windows containers. If false, switches Docker Desktop to toggle the Docker CLI to talk to the Linux daemon, enabling Linux containers. This overrides anything that may have been set at installation using the `--no-windows-containers` flag.|
+| `linuxVM` | Parameters and settings related to Linux VM options - grouped together here for convenience. |
+|`wslEngineEnabled`  |<span class="badge badge-info">Windows only</span> If `value` is set to true, Docker Desktop uses the WSL 2 based engine. This overrides anything that may have been set at installation using the `--backend=<backend name>` flag. It is also incompatible with Enhanced Container Isolation. See [Known issues](faq.md) for more information.|
```

Comment:
We need to more clearly denote that `wslEngineEnabled`, `dockerDaemonOptions` and `vpnkitCIDR` are nested values within the `linuxVM` section. The way this table [looks on  this page](https://deploy-preview-15775--docsdocker.netlify.app/desktop/hardened-desktop/admin-controls/configure-ac/) makes it seem like each of these value are at the same level in the JSON hierarchy.